design-tokens/type-family.scss

```diff
@@ -2,32 +2,40 @@
 
 @font-face {
     font-family: 'BC Sans';
-    src: url('../assets/font/BCSans-Regular.woff') format('woff'),
-        url('../assets/font/BCSans-Regular.woff2') format('woff2');
+    src: url('/node_modules/@bcgov-nr/nr-fsa-theme/assets/font/BCSans-Regular.woff')
```

Comment:
in the real world when we use this file, the path for these font files are under the node module package